bot/cogs/information.py

```diff
@@ -146,15 +124,15 @@ def __init__(self, bot: Bot):
                 **Counts**
                 Members: {member_count:,}
                 Roles: {roles}
-                Text: {text_channels}
-                Voice: {voice_channels}
-                Channel categories: {category_channels}
+                Text Channels: {channels[TextChannel]}
+                Voice Channels: {channels[VoiceChannel]}
+                Channel categories: {channels[CategoryChannel]}
```

Comment:
With the `ChannelType` suggestions, this description would not be accounting for all types (most significant is the `news` type currently) or any future types that Discord may introduce. I could live with that, but if you're up for it, you could make this part of the description be dynamically generated from whatever types are inside the `channels` dictionary.

Reply:
Call me dumb, but I'm not sure if I exactly understand what you mean here. I assume you basically mean something like this? 
```py
        channel_counts = ""
        for ch in channels:
            channel_counts += f"{str(ch).title()} channels: {channels[ch]} \n"
```
If so, that for-loop can be simplified using this. Would this be preferred instead?
```py
        channel_counts = "".join(f"{str(ch).title()} channels: {channels[ch]} \n" for ch in channels)
```
All of the changes have been made, I just wanna get this with the commit. :)

On my own testing server, this yields:
```
Voice channels: 11
Category channels: 8
Text channels: 41
```
We could sort it alphabetically too, if that is wanted.

Reply:
Yes, that is what I meant. Sorting it sounds like a good idea too.

Reply:
Also, it's perfectly fine, and perhaps even better, for this change to have been a separate commit. No need to wait.